rocketmq-store/src/config/flush_disk_type.rs

```diff
@@ -28,6 +28,15 @@ pub enum FlushDiskType {
     AsyncFlush,
 }
 
+impl FlushDiskType {
+    pub fn get_flush_disk_type(&self) -> &'static str {
+        match self {
+            FlushDiskType::SyncFlush => "SyncFlush",
+            FlushDiskType::AsyncFlush => "AsyncFlush",
+        }
+    }
+}
+
```

Comment:
Due to the differences in naming conventions between Rust and Java, this particular aspect should align with Java's standards.

'''
    SYNC_FLUSH,
    ASYNC_FLUSH

'''